### PR TITLE
bugfix: make sure symbol loading from Base.gc_page_utilization_data works on linux

### PR DESCRIPTION
**Pull request by d-netto**

### CODE REVIEW

File: src/jl_exported_data.inc [hunk 1]
```diff
@@ -148,6 +148,7 @@
     XX(jl_libdl_module) \
     XX(jl_libdl_dlopen_func) \
     XX(jl_precompilable_error) \
+    XX(jl_gc_page_utilization_stats) \
```
Comment (vtjnash) [inline src/jl_exported_data.inc:151]:
```suggestion
```

File: src/gc.c [hunk 2]
```diff
@@ -3819,6 +3819,7 @@ void jl_gc_init(void)
     gc_num.allocd = 0;
     gc_num.max_pause = 0;
     gc_num.max_memory = 0;
+    jl_gc_page_utilization_stats = (double *)malloc_s(JL_GC_N_MAX_POOLS * sizeof(double));
```
Comment (vtjnash) [inline src/gc.c:3822]:
```suggestion
```

File: src/gc.c [hunk 3]
```diff
@@ -1423,7 +1423,7 @@ int jl_gc_classify_pools(size_t sz, int *osize)
 // sweep phase
 
 gc_fragmentation_stat_t gc_page_fragmentation_stats[JL_GC_N_POOLS];
-JL_DLLEXPORT double gc_page_utilization_stats[JL_GC_N_MAX_POOLS];
+JL_DLLEXPORT double *jl_gc_page_utilization_stats;
```
Comment (vtjnash) [inline src/gc.c:1426]:
```suggestion
JL_DLLEXPORT double jl_gc_page_utilization_stats[JL_GC_N_MAX_POOLS];
```

Comment (vtjnash) [reply src/gc.c:1426]: you shouldn't need to export a pointer to it, just itself

**Comment by vchuravy** [comment]
Could you also add a test (probably `test/gc.jl`)?

**Comment by d-netto** [comment]
Will do.

**Pull request merged**